civis/io/_tables.py

```diff
@@ -694,21 +689,26 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
         warnings.warn("`archive` is deprecated and will be removed in v2.0.0. "
                       "Use `hidden` instead.", FutureWarning)
 
+    delimiter = '|'
```

Comment:
Pipe is a better delimiter than a comma, but since we're writing out the csv and pandas is doing the quoting of special characters I'm not sure it matters. I might just use the default value, but no harm in using a pipe.

Reply:
I'd have to go back and look but anecdotally, I've only had consistent success importing using a pipe. Some of that may be tied in with the backslash quote issue I mentioned above so I'd have to write up some test cases to see if there's a real difference.

In any case, it's the preferred delimiter in our [import documentation](https://civis.zendesk.com/hc/en-us/articles/213756143-File-Import).